tests/src/Content/Text/HTMLTest.php

```diff
@@ -58,6 +58,10 @@ public function dataHTMLText()
 				'expectedBBCode' => '[audio]http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3[/audio]',
 				'html' => '<audio src="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3" controls="controls"><a href="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3</a></audio>',
 			],
+			'bug-8075-html-tags' => [
+				'expectedBBCode' => "&lt;big rant here&rt; I don't understand tests",
+				'html' => "<big rant here> I don't understand tests",
```

Comment:
No, it's the opposite, the `html` property should have the entities, the `expectedBBCode` shouldn't!

Reply:
Gnarf ...